controllers/notes.controller.js

```diff
@@ -0,0 +1,59 @@
+const { Note } = require('../models/note.model');
+const { extend } = require('lodash');
+
+const createNoteForVideo = async (req, res) => {
+	try {
```

Comment:
Can use the async handler which was discussed in the ask-tanay-discussions by purva and tanay.